en/docs/install-and-setup/install/installing-the-product/installing-mi-dashboard.md

```diff
@@ -7,6 +7,19 @@ Follow the steps given below to install the Micro Integrator (MI) Dashboard.
 1.  Go to the [WSO2 Micro Integrator web page](https://wso2.com/integration/micro-integrator/#), click **Download**, and then click **Other Resources/MI Dashboard** to download the MI Dashboard as a ZIP file.
 2.  Extract the archive file to a dedicated directory for the Micro Integrator Dashboard, which will hereafter be referred to as `<MI-DASHBOARD_HOME>`.
 
+!!! info
+    To connect the MI servers with the dashboard, add the following configuration to the `deployment.toml` file (stored in the `<MI_HOME>/conf/` folder of each server instance.
+    ```
+    [dashboard_config]
+    dashboard_url = "https://{hostname/ip}:{port}/dashboard/api/"
```

Comment:
Here, **dashboard_url** is the only **mandatory** parameter. Others are optional. Shall we add that detail too?

Reply:
The configs and the details are described in the given link [1]. Since it is redundant to duplicate that info, I did not add it.

[1] https://apim.docs.wso2.com/en/latest/observe/mi-observe/working-with-monitoring-dashboard/#step-2-configure-the-mi-servers